ansible/tests/redpanda-kafka-api-1thread.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "redpanda-1thread",
+    "mq_type": "Kafka",
```

Comment:
Redpanda

Reply:
In this case we're reusing kafka sender / receiver as Redpanda api is kafka compatible